### PR TITLE
Prefer choco over manual installation for windows

### PR DESCRIPTION
**Pull request by awesley**
As a Windows developer who benefits from Chocolatey's wide adoption, I propose making this the primary example of Windows installation.  It is my suspicion that most people using Minikube on Windows will be familiar with it (though I have no data to be back up this claim).

### CODE REVIEW

**Comment by dlorenc** [comment]
Closing, please reopen if you can sign the CLA.

**Comment by awesley** [comment]
I have done so now.

File: README.md [hunk 1]
```diff
@@ -31,7 +31,9 @@ curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/miniku
 ```
 
 ### Windows
-Download the [minikube-windows-amd64.exe](https://storage.googleapis.com/minikube/releases/latest/minikube-windows-amd64.exe) file, rename it to `minikube.exe` and add it to your path.
+```shell
+choco install minikube
+```
```
Comment (dlorenc) [inline README.md]: Could you keep both sets of instructions?

Comment (awesley) [reply README.md]: @dlorenc Done

**Pull request merged**